docs/install_linux_mac.rst

```diff
@@ -170,13 +170,14 @@ one-by-one:
 
 .. code-block:: none
 
-    brew install python --with-brewed-openssl
+    brew install python@3.8
     brew install git
-    brew tap caskroom/versions
-    brew cask install homebrew/cask-versions/adoptopenjdk11
+    brew cask install AdoptOpenJDK/openjdk/adoptopenjdk11
```

Comment:
This should be all lowercase.
`brew cask install adoptopenjdk/openjdk/adoptopenjdk11`